src/pages/VideoContentPage/VideoContentPage.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router';
+import IcBaselineAddTask from '../../components/UI/Icons/IcBaselineAddTask';
+import IcOutlineThumbUp from '../../components/UI/Icons/IcOutlineThumbUp';
+import IcRoundPlaylistAdd from '../../components/UI/Icons/IcRoundPlaylistAdd';
+import IcRoundThumbDownOffAlt from '../../components/UI/Icons/IcRoundThumbDownOffAlt';
+import ViewCount from '../../utils/ViewCount.jsx';
+import './VideoContent.css';
+
+function VideoContentPage() {
+  const location = useLocation();
+  const [VideoUrl, setVideoUrl] = useState();
+    const data = location.state;
+    console.log(data);
+    const videosrc = "https://www.youtube.com/watch?v=" + data.videoid;
+  const videosrc1 = `https://www.youtube.com/watch?v=${data.videoid}`;
```

Comment:
Any particular reason to keep both of these? Please look into it and remove one of them otherwise.

Reply:
no particular reason i was trying a approach didn't get the results so now no use of both these variables will remove them